src/util/FactorUtil.js

```diff
@@ -139,6 +145,9 @@ function (Okta, TimeUtil) {
     if (provider === 'OKTA' && factorType === 'push') {
       return 'OKTA_VERIFY_PUSH';
     }
+    if (provider === 'CUSTOM' && factorType === 'token:software:hotp') {
```

Comment:
Same question as above.. Can we leave out provider check?

Reply:
yeah can omit this.